wcfsetup/install/files/js/WoltLabSuite/Core/StringUtil.js

```diff
@@ -96,8 +96,45 @@ define(['Language', './NumberUtil'], function(Language, NumberUtil) {
 		 * @param	{?}		string
 		 * @return	{String}
 		 */
-		unescapeHTML: function (string) {
+		unescapeHTML: function(string) {
 			return String(string).replace(/&amp;/g, '&').replace(/&quot;/g, '"').replace(/&lt;/g, '<').replace(/&gt;/g, '>');
+		},
+		
+		/**
+		 * Shortens numbers larger than 1000 by using unit prefixes.
+		 *
+		 * @param	{?}		number
+		 */
+		shortUnit: function(number) {
+			var unitSuffix = '';
+			
+			if (number >= 1000000) {
+				number /= 1000000;
+				
+				if (number > 10) {
+					number = Math.floor(number);
+				}
+				else {
+					number = NumberUtil.round(number, -1);
+				}
+				
+				unitSuffix = 'M';
+			}
+			else if (number >= 1000) {
+				number /= 1000;
+				
+				if (number > 10) {
+					number = Math.floor(number);
+				}
+				else {
+					number = NumberUtil.round(number, -1);
+					console.log(number);
+				}
+				
+				unitSuffix = 'k';
+			}
+			
+			return this.formatNumeric(number, -1) + unitSuffix;
```

Comment:
Is the `-1` necessary at this place? It's missing in the PHP version of `shortUnit`.